integrations/slack/slack.go

```diff
@@ -0,0 +1,130 @@
+package slack
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+)
+
+// postMessage API URL
+const URI = "https://api.slack.com/api/chat.postMessage"
```

Comment:
Instead of hardcoding this, we can let the user calling the `NewClient` method to pass the Slack API URL as well.